pkg/asset/installconfig/openstack/validate.go

```diff
@@ -47,3 +53,9 @@ func ValidateForProvisioning(ic *types.InstallConfig) error {
 	}
 	return nil
 }
+
+func defaultOpenStackMachinePoolPlatform() openstack.MachinePool {
+	return openstack.MachinePool{
+		Zones: []string{""},
```

Comment:
This is really an awful default. Please remind me why we do this?

Reply:
@mandre Because of this https://github.com/openshift/installer/blob/master/pkg/asset/machines/worker.go#L112-L116

Reply:
That doesn't explain why the default Zones is a slice with an empty element. I'd expect it to be an empty slice but surely I'm missing something.

Reply:
This default is bothering me but changing it isn't in scope for this PR.

According to @iamemilio, its a slice with an empty string because passing an empty string to zones is how you set it to use Nova default.